docs/data/base/components/snackbar/snackbar.md

```diff
@@ -0,0 +1,145 @@
+---
+product: base
+title: Unstyled React Snackbar component and hook
+components: SnackbarUnstyled
+githubLabel: 'component: snackbar'
+---
+
+# Unstyled snackbar
+
+<p class="description">The SnackbarUnstyled component informs users that an action has been or will be performed by the app.</p>
+
+## Introduction
+
+A snackbar provides users with a brief, temporary message about app processes without interrupting their activity or experience.
+
+The `SnackbarUnstyled` component is built to appear in the corner or center of the screen to inform users about an action that the app is taking.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Component
+
+### Usage
+
+After [installation](/base/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
+import SnackbarUnstyled from '@mui/base/SnackbarUnstyled';
+
+export default function MyApp() {
+  return <SnackbarUnstyled>{/* snackbar text */}</SnackbarUnstyled>;
+}
+```
+
+### Basics
+
+:::info
+Snackbars are differentiated from other kinds of popups and alerts in that they are _informational, temporary, and contextual_ (Source: [Material Design—Snackbars](https://material.io/components/snackbars)).
```

Comment:
Could be maybe not link Material Design, it may be misleading.

Reply:
After further reflection, I agree now, we can keep the text and remove the link. @samuelsycamore was a bit hesistant to use Material Design as a source for the definition of a snackbar. See https://github.com/mui/material-ui/pull/33227#issuecomment-1218349038. 

Perhaps we may want to alter the following line as well (which used MD as a source), since SnackbarUnstyled can actually be placed anywhere on the screen: 
> The SnackbarUnstyled component is built to appear in the corner or center of the screen to inform users about an action that the app is taking.

Don't mention `center` or `corner`.

@mnajdova What do you think? 

I'll leave it upto @samuelsycamore .

Reply:
Cool, I'll revise this a little to remove the MD specs and source.

Reply:
@samuelsycamore Will you be continuing on this PR or will do it later to unblock the merge?

Reply:
Sorry for dropping the ball here! I've removed the MD callout as well as the mention of position for the snackbar.